Tasks/Npm/npmtask.ts

```diff
@@ -87,6 +97,16 @@ async function runNpmAuthHelperAsync(npmAuthRunner: trm.ToolRunner) : Promise<nu
     }
 }
 
+function getUserNpmrcPath() {
+    var userNpmRc = process.env['npm_config_userconfig'];
+    if(!userNpmRc){
+        // default npm rc is located at user's home folder.
+        userNpmRc = path.join(process.env['HOMEDRIVE'], process.env['HOMEPATH'], '.npmrc');
```

Comment:
We're going to have to worry about non-windows eventually, and frankly I think it's ... *unfortunate* that we didn't do it to begin with